aws/resource_aws_emr_instance_group.go

```diff
@@ -177,6 +200,19 @@ func resourceAwsEMRInstanceGroupRead(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("error reading EMR Instance Group (%s): %s", d.Id(), err)
 	}
 
+	switch {
+	case len(ig.Configurations) > 0:
+		configOut, err := flattenConfigurationJson(ig.Configurations)
```

Comment:
Totally not for this pull request, but seems like it would be nice if `flattenConfigurationJson` just did

```go
if len(config) == 0 {
  return "", nil
}
```

so we didn't need the surrounding conditional logic. 👍